package.json

```diff
@@ -15,6 +15,7 @@
     "axios": "*",
     "estimo": "^2.2.9",
     "isomorphic-fetch": "2.2.1",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
Should also bump the patch version of our package

Reply:
Done